### PR TITLE
Update volumio.markdown

### PR DESCRIPTION
**Pull request by Rapace29**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/volumio.markdown [hunk 1]
```diff
@@ -37,4 +37,109 @@ port:
...
+media_artist:
+  Artist of current playing media (Music track only).
+  required: false 
+  type: string
+media_album_name:
+  Artist of current playing media (Music track only).
+  required: false 
+  type: string
+media_image_url:
+  Image url of current playing media.
```
Comment (Rapace29) [inline source/_integrations/volumio.markdown]: Image URL of current playing media.

**Review by Rapace29 — COMMENTED** [review]
URL au lieu de url

**Review by Rapace29 — COMMENTED** [review]
check errors

**Comment by Rapace29** [comment]
check errors

**Comment by Rapace29** [comment]
I don't understand the error  in line 24 column 11

File: source/_integrations/volumio.markdown [hunk 2]
```diff
@@ -37,4 +37,109 @@ port:
...
+async_media_play:
+  Send media_play command to media player.
+  required: false 
+  type: string
+async_set_volume_level:
+  Send volume_up command to media player.
+  required: false 
+  type: integer
+async_volume_up:
+  Service to send the Volumio the command for volume up."
```
Comment (Rapace29) [inline source/_integrations/volumio.markdown:118]: Service to send the Volumio the command for volume up.

**Comment by klaasnicolaas** [comment]
??

**Pull request closed without merging**